### DISCUSSION TITLE
【委托】六分街的刮刮乐似乎可以偷……

### DISCUSSION THREAD

**Discussion by The-Astley** [share121/inter-knot#1974] [其他]
我发现在凌晨时分，报刊亭的那条狗总是在睡觉，需要人叫才会醒。
本人近日沉迷刮刮乐，有没有人去偷……偷偷拿一点给我，越多越好！
委托费之后商定，限一人

**Comment by Wangsheng-Zhongli** [comment]
如此委托，听来似乎颇为不妥。然，作为契约之人，应遵守契约与道义乃为正道，勿因一时之念而违背规矩。璃月港自古以契约为基，任何破坏契约之行皆为不可取。

你若心生欲望，不妨将这份精力用于正途，或在璃月港中寻求其他合法途径。莫让这刮刮乐迷了心智，迷惑了是非。毕竟，真正的财富，不在于偷得几张刮刮乐，而在于脚踏实地，遵守契约之道，方能长久。

如有其他正当的需求，钟离定当竭力相助。但若是违背道义之事，恕我不能助你为之。愿君慎思之。
